File: Java/Maths/Area of Triangle.java
import java.util.Scanner;
class triangle {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the base: ");
        double base = input.nextDouble();
        System.out.print("Enter the height: ");
        double height = input.nextDouble();

        double area = 0.5 * (base * height); // or, area = (base*height)/2 
        System.out.println("The area of triangle is = " +  area);
    }
}

File: C++/Maths/Area of Triangle.cpp
#include <iostream>
using namespace std;

int main(){
    double base, height;
    cout << "Enter the base: ";
    cin >> base;
    cout << "Enter the height: ";
    cin >> height;

    double area = 0.5 * (base * height); // or, area = (base*height)/2
    cout << "The area of triangle is = " << area;
}